### PR TITLE
[ch26532]: Recently Used emojis are updated only after app reopening

### PR DESCRIPTION
**Pull request by Strafi**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "emoji-picker-react",
-  "version": "3.4.5",
+  "version": "3.4.6",
```
Comment (VictoriaTheBrave) [inline package.json:3]: Shouldn't we have a PR for fullstack with updated version of this package?

**Review by VictoriaTheBrave — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Strafi) [reply package.json:3]: Of course we should, I will open it after this will be merged

File: src/lib/reducer/index.js [hunk 2]
```diff
@@ -67,6 +69,11 @@ const reducer = (state, { type, ...action }) => {
         ...state,
         ...action,
       };
+	case actionTypes.UPDATE_RECENTLY_USED:
+      return {
+        ...state,
+		recentlyUsed: getRecentlyUsed(),
```
Comment (Crypto-kittensSeller) [inline src/lib/reducer/index.js:75]: indentation

File: src/lib/reducer/index.js [hunk 3]
```diff
@@ -67,6 +69,11 @@ const reducer = (state, { type, ...action }) => {
         ...state,
         ...action,
       };
+	case actionTypes.UPDATE_RECENTLY_USED:
+      return {
```
Comment (Crypto-kittensSeller) [inline src/lib/reducer/index.js:73]: indentation

Comment (Crypto-kittensSeller) [inline src/lib/reducer/index.js:73]: indentation

**Review by Crypto-kittensSeller — APPROVED** [review]

**Pull request merged**